Mixpanel/Mixpanel.m

```diff
@@ -254,9 +254,10 @@ + (void)assertPropertyTypesInArray:(NSArray *)arrayOfProperties depth:(NSUIntege
 
 - (NSString *)defaultDistinctId
 {    
-    NSString *distinctId = [self IFA];
+    NSString *distinctId;
 
 #if !defined(MIXPANEL_WATCHOS) && !defined(MIXPANEL_MACOS)
+    distinctId = [self IFA];
```

Comment:
Well, no, that's the warning. You assign it a value, and immediately assign it another value so the first assignment has no effect.

In the olden days I would have initialized it to nil, but that is no longer necessary in the modern runtime IIRC.